05.ls/ls.rb

```diff
@@ -123,27 +123,27 @@ def output_detail(detail_files)
 
 def output_normal(all_files)
   row_count = (all_files.size.to_f / MAX_COLUMN).ceil
-  display_width = adjust_width(all_files).max
+  col_width = calc_normal_col_width(all_files).max
```

Comment:
`calc_normal_col_width`を呼ぶと「幅はこれにしなよ」と整数値が返ってくるメソッド設計の方がシンプルだと思いました。

```suggestion
  col_width = calc_normal_col_width(all_files)
```

加えて、そうした場合、

- calc_normal_col_width は整数値を返す
- calc_detail_col_width はハッシュを返す

と、戻り値の型が異なるので、たとえば後者は`calc_detail_col_widths`のようにsを付けて、複数の値（ここではハッシュ）が返るよ、というのをほのめかすとより親切かもしれません